ddspipe_core/src/cpp/communication/dds/Track.cpp

```diff
@@ -218,9 +219,9 @@ void Track::transmit_() noexcept
 
         // Get data received (send empty data to be created(allocated) in reader)
         std::unique_ptr<IRoutingData> data;
-        utils::ReturnCode ret = reader_->take(data);
+        auto ret = reader_->take(data);
```

Comment:
I think it's better to make a explicit reference to the ReturnCode class for improving code understanding. 
```suggestion
        utils::ReturnCode ret = reader_->take(data);
```